### ISSUE TITLE
Incompatible with Earthbounds+Better combat

### ISSUE THREAD

**Issue by Rad586** [GekidoLukas/weaponleveling#25]
**Describe the bug**
Select a weapon(in a mod) that has high attack speed, hold the attack(left mouse button) key, go attack something, game crash.
**Mod/Minecraft Version**
Minecraft Version: 1.18.2 fabric
Mod Version: 1.4.2

Mod list:
1.geckolib-fabric-1.18-3.0.59
2.TerraBlender-fabric-1.18.2-1.2.0.126
3.The_Graveyard_1.9_(FABRIC)_for_1.18.2 (just for it's bone dagger)
4.bettercombat-fabric-1.4.5+1.18.2
5.cloth-config-6.4.90-fabric
6.fabric-api-0.66.0+1.18.2
7.ForgeConfigAPIPort-v3.2.4-1.18.2-Fabric
8.player-animation-lib-fabric-0.4.0-test4+1.18
9.weaponleveling-fabric-1.18.2-1.4.2

**To Reproduce**
1. Select a melee weapon in mod(any) that has high attack speed
2. Hold the attack key for constantly swinging
3. Attack a mob in status mentioned above
4. Game crash

**Expected behavior**
In short, fix it. 
Actually I don't think this should be treated as first priority, but for modpack developers, they are likely to have these three mods together...

**Additional context**
That's kinda ... to find such a bug, right?

crash-report:
[crash-2022-11-20_15.04.43-server.txt](https://github.com/GeradesoLukas/weaponleveling/files/10049061/crash-2022-11-20_15.04.43-server.txt)

**Comment by GekidoLukas** [comment]
I cannot reproduce this issue and will therefore close it
